client/src/views/Create.vue

```diff
@@ -70,6 +72,10 @@ export default {
           component: HODForm,
           model: 'hod',
         },
+        {
+          component: TracerProfileForm,
+          model: 'tracerProfile',
+        },
```

Comment:
Per initial_state.json this should have tracer_profile as model. Could run into issues with overwrites during unpacking if naming convention isn't exact.

Reply:
Yep, I missed that. Thank you.